packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,201 @@
+    }
+  }, []);
+
+  const [error, setError] = useState('');
+  const [password, setPassword] = useState('');
+  const useLoggedIn = () => {};
+
+  const signInWithPassword = () => {};
+
+  // TODO:
+  // Add in the Alert Bar in place of the original `success` and `error` display divs
+
+  return (
+    <>
+      <div className="mb-4">
+        {isPasswordNeeded ? (
+          <FtlMsg id="signin-password-needed-header">
+            <h1 className="card-header">
+              Enter your password{' '}
+              <span className="card-subheader">for your Firefox account</span>
+            </h1>
+          </FtlMsg>
+        ) : (
+          <>
+            <FtlMsg id="signin">
+              <h1 className="card-header">Sign in</h1>
+            </FtlMsg>
+            <>
+              {serviceLogo ? (
+                <span className="card-subheader">
+                  <FtlMsg id="signin-subheader-with-logo">Continue to </FtlMsg>
+                  {serviceLogo}
+                </span>
+              ) : (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          <div className="avatar-wrapper mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
+          <div className="my-5 text-base break-all">{email}</div>
+        </div>
+        <form noValidate {...{ onSubmit }}>
+          <input type="email" className="email hidden" value={email} disabled />
+
+          {isPasswordNeeded && (
+            <InputPassword
+              className="mb-2"
+              label={localizedPasswordFormLabel}
+              hasErrors={error.length > 0}
+              errorText={error}
+              tooltipPosition="bottom"
+              required={true}
+              autoFocus={true}
+              onChange={(e) => setPassword(e.currentTarget.value)}
+            />
+          )}
+          {/* This non-fulfilled input tricks the browser, when trying to
+              sign in with the wrong password, into not showing the doorhanger.
+           */}
+          <input className="hidden" required />
+
+          <div className="flex">
+            <button
+              className="cta-primary cta-xl"
+              type="submit"
+              onClick={isPasswordNeeded ? useLoggedIn : signInWithPassword}
+            >
+              <FtlMsg id="signin">Sign in</FtlMsg>
+            </button>
+          </div>
+        </form>
+        <div id="tos-pp" className="text-grey-500 my-5 text-xs">
+          {isPocketClient ? (
+            <FtlMsg id="firefox-and-pocket-tos">
+              <p className="text-xs">
```

Comment:
Should these links use `LinkExternal` as done in the fxa-react footer?

Reply:
OH YEAH sorry yes definitely